### PR TITLE
Blog/ Cloud Optimization: How Sourcegraph Helps

### LINKED ISSUE #5607: Create and publish the Cloud optimization blog post
Stakeholder:
What? Create and publish a new blog post on how to reduce cloud costs
Why?

Assets:
* Draft, including meta: https://docs.google.com/document/d/1tAq25YIC-dvIo3NfSP7vdT36l9JIhtbeCci_Y2GOXZQ/edit#

Asana: https://app.asana.com/0/1200095757619647/1202662087404186/f

### PR DESCRIPTION
**Pull request by st0nebraker**
Closes #5607 by adding a new blogpost ([draft here](https://docs.google.com/document/d/1tAq25YIC-dvIo3NfSP7vdT36l9JIhtbeCci_Y2GOXZQ/edit#)).

### Test
1. Ensure prettier has standardized the proposed changes.
2. Navigate to [`/blog/cloud-optimization-part-1-visibility`](https://deploy-preview-5617--sourcegraph.netlify.app/blog/cloud-optimization-part-1-visibility) and ensure requirements are met.

### CODE REVIEW

**Review by bretthayes — CHANGES_REQUESTED** [review]
Blog post content looks good! Could you compress those png assets a bit more? I think we can do a lot better and reduce most of the file sizes by 75%.

**Comment by st0nebraker** [comment]
> Blog post content looks good! Could you compress those png assets a bit more? I think we can do a lot better and reduce most of the file sizes by 75%.

Sure thing! Check it out now

**Review by bretthayes — CHANGES_REQUESTED** [review]
@st0nebraker looks like you may have to update the directory name where the assets live since the title of the slug changed. The images in the Figure components don't seem to be loading in the preview.

Btw, the assets look awesome! Thanks for compressing 👏🏻

**Comment by st0nebraker** [comment]
> @st0nebraker looks like you may have to update the directory name where the assets live since the title of the slug changed. The images in the Figure components don't seem to be loading in the preview.
> 
> Btw, the assets look awesome! Thanks for compressing 👏🏻

Thanks and good catch! Try now

**Review by bretthayes — APPROVED** [review]

File: content/blogposts/2022/cloud-optimization-part-1-visibility.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+<Figure 
+  src="/blog/cloud-optimization-part-1-visibility/google-cloud-settings-form.png"
+  alt="Google cloud settings form"
+  caption="Creating a new GCP VM using the GCP web console (https://console.cloud.google.com/compute/instancesAdd)"
```
Comment (christinaforney) [inline content/blogposts/2022/cloud-optimization-part-1-visibility.md:31]: @bretthayes - do we have the ability to hyperlink from a caption?

File: content/blogposts/2022/cloud-optimization-part-1-visibility.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+### Infrastructure as code 
+Infrastructure as code (IaC) is the process of managing and provisioning cloud infrastructure through machine-readable definition files like Terraform ([open source repositories with examples](https://sourcegraph.com/search?q=context:global+lang:Terraform+select:repo&patternType=literal), [example files](https://sourcegraph.com/search?q=context:global+lang:Terraform+&patternType=literal)) and AWS CloudFormation ([open source repositories with examples](https://sourcegraph.com/search?q=context:global+select:repo+AWSTemplateFormatVersion&patternType=literal), [example files](https://sourcegraph.com/search?q=context:global+AWSTemplateFormatVersion&patternType=literal)), rather than physical hardware configuration or interactive configuration tools [1]. These definition files make it possible to have visibility of all your provisioned infrastructure, especially if you have a [universal code search solution](http://sourcegraph.com) inside your organization.
```
Comment (christinaforney) [inline content/blogposts/2022/cloud-optimization-part-1-visibility.md:26]: @bretthayes For the footnote, is there a way to hyperlink to the footer?

File: content/blogposts/2022/cloud-optimization-part-1-visibility.md (hunk 1 above)
Comment (st0nebraker) [reply content/blogposts/2022/cloud-optimization-part-1-visibility.md]: Right now we can link the whole caption, but not parts of it yet. I added this to this blog in [f2f88cb](https://github.com/sourcegraph/about/pull/5617/commits/f2f88cbbba8455a8f37a5d808b887e67633bdfc9)!

File: content/blogposts/2022/cloud-optimization-part-1-visibility.md (hunk 2 above)
Comment (st0nebraker) [reply content/blogposts/2022/cloud-optimization-part-1-visibility.md]: Added it in this commit ([840c151](https://github.com/sourcegraph/about/pull/5617/commits/840c15121f476dd49df321ae1d7547e47585f800)) for you! 😄

**Review by christinaforney — APPROVED** [review]
Woo hoo! Looks great! Thanks @st0nebraker

**Pull request merged**